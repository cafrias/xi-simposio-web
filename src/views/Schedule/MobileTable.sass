@import '../../_sass/_constants.sass'
@import '../../_sass/_breakpoints.sass'

.mobile-table
  width: 100%
  border-collapse: collapse
  margin-bottom: 2.618rem

// TABLE BASICS
.table__date
  font-size: 2.023rem
  color: $secondary

.table__weekday
  display: block
  color: $dark-brown
  font-size: 1.618rem

// TABLE ACTIVITY
.activity-row>td
  border-top: 1px solid $light-brown
  padding: 1rem 0

.activity__name
  text-align: center

.activity__time
  font-weight: 900
  color: $dark-brown
