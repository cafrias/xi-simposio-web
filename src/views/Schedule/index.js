// @flow

import React from 'react'

import './Schedule.sass'

// COMPONENTS __________________________________________________________________
import Section from '../../components/Section'
import MobileTable from './MobileTable'

// DATA ________________________________________________________________________
import days from './data'

function Schedule() {
    return(
      <Section id="schedule" name="Cronograma" className="schedule">
        <div className="mobile-content text-container">
          {
            days.map((day, i) => (
              <MobileTable key={i} {...day}/>
            ))
          }
        </div>
      </Section>
    )
}

export default Schedule
