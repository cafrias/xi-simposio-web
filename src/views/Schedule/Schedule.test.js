import React from 'react'
import {shallow} from 'enzyme'

import Schedule from './index'

describe('Schedule view', () => {
  var wrapper

  beforeEach(() => wrapper = shallow(<Schedule />))

  it('should render without errors', () => {
    expect(wrapper).toHaveLength(1)
  })

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
