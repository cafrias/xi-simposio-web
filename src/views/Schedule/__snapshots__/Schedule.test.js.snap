exports[`Schedule view should match snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Section
    className="background_white schedule"
    id="schedule"
    name="Cronograma">
    <div
        className="mobile-content text-container">
        <MobileTable
            activities={
                Object {
                    "15.00": "Acreditaciones",
                    "17.00": "Acto de Apertura",
                    "17.30": "Trabajo en comisiones",
                    "19.00": "Coffee Break",
                    "19.30": "Conferencia",
                    "21.00": "Cóctel de bienvenida",
                  }
            }
            date="17/05/2017"
            weekday="Miércoles" />
        <MobileTable
            activities={
                Object {
                    "17.00": "Trabajo en comisiones",
                    "17.30": "Trabajo en comisiones",
                    "19.00": "Coffee Break",
                    "19.30": "Conferencia",
                    "21.00": "Cena libre",
                  }
            }
            date="18/05/2017"
            weekday="Jueves" />
        <MobileTable
            activities={
                Object {
                    "17.00": "Trabajo en comisiones",
                    "17.30": "Trabajo en comisiones",
                    "19.00": "Coffee Break",
                    "19.30": "Conferencia",
                    "21.00": "Acto de clausura",
                    "22.30": "Cena de camaradería",
                  }
            }
            date="19/05/2017"
            weekday="Viernes" />
    </div>
    <div
        className="desktop-content">
        <DesktopTable
            days={
                Array [
                    Object {
                      "activities": Object {
                        "15.00": "Acreditaciones",
                        "17.00": "Acto de Apertura",
                        "17.30": "Trabajo en comisiones",
                        "19.00": "Coffee Break",
                        "19.30": "Conferencia",
                        "21.00": "Cóctel de bienvenida",
                      },
                      "date": "17/05/2017",
                      "weekday": "Miércoles",
                    },
                    Object {
                      "activities": Object {
                        "17.00": "Trabajo en comisiones",
                        "17.30": "Trabajo en comisiones",
                        "19.00": "Coffee Break",
                        "19.30": "Conferencia",
                        "21.00": "Cena libre",
                      },
                      "date": "18/05/2017",
                      "weekday": "Jueves",
                    },
                    Object {
                      "activities": Object {
                        "17.00": "Trabajo en comisiones",
                        "17.30": "Trabajo en comisiones",
                        "19.00": "Coffee Break",
                        "19.30": "Conferencia",
                        "21.00": "Acto de clausura",
                        "22.30": "Cena de camaradería",
                      },
                      "date": "19/05/2017",
                      "weekday": "Viernes",
                    },
                  ]
            } />
    </div>
</Section>,
  "nodes": Array [
    <Section
      className="background_white schedule"
      id="schedule"
      name="Cronograma">
      <div
            className="mobile-content text-container">
            <MobileTable
                  activities={
                        Object {
                              "15.00": "Acreditaciones",
                              "17.00": "Acto de Apertura",
                              "17.30": "Trabajo en comisiones",
                              "19.00": "Coffee Break",
                              "19.30": "Conferencia",
                              "21.00": "Cóctel de bienvenida",
                            }
                  }
                  date="17/05/2017"
                  weekday="Miércoles" />
            <MobileTable
                  activities={
                        Object {
                              "17.00": "Trabajo en comisiones",
                              "17.30": "Trabajo en comisiones",
                              "19.00": "Coffee Break",
                              "19.30": "Conferencia",
                              "21.00": "Cena libre",
                            }
                  }
                  date="18/05/2017"
                  weekday="Jueves" />
            <MobileTable
                  activities={
                        Object {
                              "17.00": "Trabajo en comisiones",
                              "17.30": "Trabajo en comisiones",
                              "19.00": "Coffee Break",
                              "19.30": "Conferencia",
                              "21.00": "Acto de clausura",
                              "22.30": "Cena de camaradería",
                            }
                  }
                  date="19/05/2017"
                  weekday="Viernes" />
      </div>
      <div
            className="desktop-content">
            <DesktopTable
                  days={
                        Array [
                              Object {
                                "activities": Object {
                                  "15.00": "Acreditaciones",
                                  "17.00": "Acto de Apertura",
                                  "17.30": "Trabajo en comisiones",
                                  "19.00": "Coffee Break",
                                  "19.30": "Conferencia",
                                  "21.00": "Cóctel de bienvenida",
                                },
                                "date": "17/05/2017",
                                "weekday": "Miércoles",
                              },
                              Object {
                                "activities": Object {
                                  "17.00": "Trabajo en comisiones",
                                  "17.30": "Trabajo en comisiones",
                                  "19.00": "Coffee Break",
                                  "19.30": "Conferencia",
                                  "21.00": "Cena libre",
                                },
                                "date": "18/05/2017",
                                "weekday": "Jueves",
                              },
                              Object {
                                "activities": Object {
                                  "17.00": "Trabajo en comisiones",
                                  "17.30": "Trabajo en comisiones",
                                  "19.00": "Coffee Break",
                                  "19.30": "Conferencia",
                                  "21.00": "Acto de clausura",
                                  "22.30": "Cena de camaradería",
                                },
                                "date": "19/05/2017",
                                "weekday": "Viernes",
                              },
                            ]
                  } />
      </div>
</Section>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Schedule />,
      "_debugID": 3,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 2,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Section
          className="background_white schedule"
          id="schedule"
          name="Cronograma">
          <div
                    className="mobile-content text-container">
                    <MobileTable
                              activities={
                                        Object {
                                                  "15.00": "Acreditaciones",
                                                  "17.00": "Acto de Apertura",
                                                  "17.30": "Trabajo en comisiones",
                                                  "19.00": "Coffee Break",
                                                  "19.30": "Conferencia",
                                                  "21.00": "Cóctel de bienvenida",
                                                }
                              }
                              date="17/05/2017"
                              weekday="Miércoles" />
                    <MobileTable
                              activities={
                                        Object {
                                                  "17.00": "Trabajo en comisiones",
                                                  "17.30": "Trabajo en comisiones",
                                                  "19.00": "Coffee Break",
                                                  "19.30": "Conferencia",
                                                  "21.00": "Cena libre",
                                                }
                              }
                              date="18/05/2017"
                              weekday="Jueves" />
                    <MobileTable
                              activities={
                                        Object {
                                                  "17.00": "Trabajo en comisiones",
                                                  "17.30": "Trabajo en comisiones",
                                                  "19.00": "Coffee Break",
                                                  "19.30": "Conferencia",
                                                  "21.00": "Acto de clausura",
                                                  "22.30": "Cena de camaradería",
                                                }
                              }
                              date="19/05/2017"
                              weekday="Viernes" />
          </div>
          <div
                    className="desktop-content">
                    <DesktopTable
                              days={
                                        Array [
                                                  Object {
                                                    "activities": Object {
                                                      "15.00": "Acreditaciones",
                                                      "17.00": "Acto de Apertura",
                                                      "17.30": "Trabajo en comisiones",
                                                      "19.00": "Coffee Break",
                                                      "19.30": "Conferencia",
                                                      "21.00": "Cóctel de bienvenida",
                                                    },
                                                    "date": "17/05/2017",
                                                    "weekday": "Miércoles",
                                                  },
                                                  Object {
                                                    "activities": Object {
                                                      "17.00": "Trabajo en comisiones",
                                                      "17.30": "Trabajo en comisiones",
                                                      "19.00": "Coffee Break",
                                                      "19.30": "Conferencia",
                                                      "21.00": "Cena libre",
                                                    },
                                                    "date": "18/05/2017",
                                                    "weekday": "Jueves",
                                                  },
                                                  Object {
                                                    "activities": Object {
                                                      "17.00": "Trabajo en comisiones",
                                                      "17.30": "Trabajo en comisiones",
                                                      "19.00": "Coffee Break",
                                                      "19.30": "Conferencia",
                                                      "21.00": "Acto de clausura",
                                                      "22.30": "Cena de camaradería",
                                                    },
                                                    "date": "19/05/2017",
                                                    "weekday": "Viernes",
                                                  },
                                                ]
                              } />
          </div>
</Section>,
        "_debugID": 4,
        "_renderedOutput": <Section
          className="background_white schedule"
          id="schedule"
          name="Cronograma">
          <div
                    className="mobile-content text-container">
                    <MobileTable
                              activities={
                                        Object {
                                                  "15.00": "Acreditaciones",
                                                  "17.00": "Acto de Apertura",
                                                  "17.30": "Trabajo en comisiones",
                                                  "19.00": "Coffee Break",
                                                  "19.30": "Conferencia",
                                                  "21.00": "Cóctel de bienvenida",
                                                }
                              }
                              date="17/05/2017"
                              weekday="Miércoles" />
                    <MobileTable
                              activities={
                                        Object {
                                                  "17.00": "Trabajo en comisiones",
                                                  "17.30": "Trabajo en comisiones",
                                                  "19.00": "Coffee Break",
                                                  "19.30": "Conferencia",
                                                  "21.00": "Cena libre",
                                                }
                              }
                              date="18/05/2017"
                              weekday="Jueves" />
                    <MobileTable
                              activities={
                                        Object {
                                                  "17.00": "Trabajo en comisiones",
                                                  "17.30": "Trabajo en comisiones",
                                                  "19.00": "Coffee Break",
                                                  "19.30": "Conferencia",
                                                  "21.00": "Acto de clausura",
                                                  "22.30": "Cena de camaradería",
                                                }
                              }
                              date="19/05/2017"
                              weekday="Viernes" />
          </div>
          <div
                    className="desktop-content">
                    <DesktopTable
                              days={
                                        Array [
                                                  Object {
                                                    "activities": Object {
                                                      "15.00": "Acreditaciones",
                                                      "17.00": "Acto de Apertura",
                                                      "17.30": "Trabajo en comisiones",
                                                      "19.00": "Coffee Break",
                                                      "19.30": "Conferencia",
                                                      "21.00": "Cóctel de bienvenida",
                                                    },
                                                    "date": "17/05/2017",
                                                    "weekday": "Miércoles",
                                                  },
                                                  Object {
                                                    "activities": Object {
                                                      "17.00": "Trabajo en comisiones",
                                                      "17.30": "Trabajo en comisiones",
                                                      "19.00": "Coffee Break",
                                                      "19.30": "Conferencia",
                                                      "21.00": "Cena libre",
                                                    },
                                                    "date": "18/05/2017",
                                                    "weekday": "Jueves",
                                                  },
                                                  Object {
                                                    "activities": Object {
                                                      "17.00": "Trabajo en comisiones",
                                                      "17.30": "Trabajo en comisiones",
                                                      "19.00": "Coffee Break",
                                                      "19.30": "Conferencia",
                                                      "21.00": "Acto de clausura",
                                                      "22.30": "Cena de camaradería",
                                                    },
                                                    "date": "19/05/2017",
                                                    "weekday": "Viernes",
                                                  },
                                                ]
                              } />
          </div>
</Section>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Schedule />,
}
`;
